packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -919,6 +919,10 @@ class MultiFrameImageStreamCompleter extends ImageStreamCompleter {
       );
       return;
     }
+    // ImageStreamCompleter disposed while waiting for next frame to be decoded.
```

Comment:
Nit: might have no listeners but still being kept alive. But there's no reason to schedule etc. If no listeners.